### PR TITLE
update schema for GET /accounts/trading-accounts

### PR DESCRIPTION
**Pull request by kenho811-b1**

### CODE REVIEW

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 1]
```diff
@@ -3077,27 +3077,56 @@ components:
...
+        - tradingAccountDescription
+        - tradingAccountId
+        - tradingAccountName
       properties:
+        isBorrowing:
+          description: Whether the trading account is borrowing a loan
+          type: boolean
+          example: false
+        isLending:
+          description: Whether the trading account is lending a loan
```
Comment (dexterhuang-blockone) [inline src/trading-api/next/v1/bullish-trading-api.yml:3096]: Whether the trading account has opened loan offers

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 2]
```diff
@@ -3077,27 +3077,56 @@ components:
...
+        - makerFee
+        - maxInitialLeverage
         - rateLimitToken
+        - takerFee
+        - tradingAccountDescription
+        - tradingAccountId
+        - tradingAccountName
       properties:
+        isBorrowing:
+          description: Whether the trading account is borrowing a loan
```
Comment (dexterhuang-blockone) [inline src/trading-api/next/v1/bullish-trading-api.yml:3092]: Whether the trading account is borrowing

Comment (dexterhuang-blockone) [reply src/trading-api/next/v1/bullish-trading-api.yml:3092]: We try to use the term **loan** only when talking about lending.

File: src/trading-api/next/v1/bullish-trading-api.yml (hunk 1 above)
Comment (kenho811-b1) [reply src/trading-api/next/v1/bullish-trading-api.yml]: I just updated it to `whether the trading account is lending` just to keep it consistent withe definition of `isBorrowing` (whether the trading_account is borrowing)

File: src/trading-api/next/v1/bullish-trading-api.yml (hunk 2 above)
Comment (kenho811-b1) [reply src/trading-api/next/v1/bullish-trading-api.yml]: THanks.

I removed the word loan.

**Review by sandeeprakhrablockone — APPROVED** [review]
👌

**Pull request merged**